extension/src/api/payment/payment.controller.js

```diff
@@ -30,12 +30,15 @@ async function processRequest(request, response) {
       paymentObject,
       authToken
     )
-
-    return httpUtils.sendResponse({
+    const result = {
       response,
       statusCode: paymentResult.success ? 200 : 400,
-      data: paymentResult.data,
-    })
+      data: paymentResult.success
+        ? { actions: paymentResult?.actions }
```

Comment:
I guess you don't need null-check here, otherwise it would fail already when you call `paymentResult.success`.

Reply:
Just checked. actions attribute depends on what handlers return, which never contain null actions.
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/869fa2d1bae80a77e668eb49419f72c2a50c657e